features/support/env.rb

```diff
@@ -68,5 +69,38 @@ def dm_notify_api_key
   ENV['DM_NOTIFY_API_KEY']
 end
 
+def dm_custom_wait_time
+  5
+end
+
 Capybara.asset_host = dm_frontend_domain
 Capybara.save_path = "reports/"
+Capybara.default_max_wait_time = 0.05
```

Comment:
Out of curiosity, what was the default before?

Reply:
2 seconds. So not dreadful, but that did make it more than 50% of our test run time 😨